mongodbatlas/clusters.go

```diff
@@ -170,6 +170,8 @@ type Cluster struct {
 
 // ProcessArgs represents the advanced configuration options for the cluster.
 type ProcessArgs struct {
+	DefaultReadConcern               string `json:"defaultReadConcern,omitempty"`
+	DefaultWriteConcern              string `json:"defaultWriteConcern,omitempty"`
```

Comment:
[q] have you tested with a number as string (`"1"`)? reasons I ask is that technically this is "majority" (string) or `1`...`n` as number, API seems to accept string but want to make sure it's handled correctly, I'll merge after confirming this

Reply:
I have not tested `defaultWriteConcern` as number, since the docs says is a string, will test `defaultWriteConcern` as number to see what happens

Reply:
@gssbzn , I changed to int64 and it seems you can pass as integer/number in request, but the issue is in the response, it will panic like this  
`
 json: cannot unmarshal string into Go struct field ProcessArgs.defaultWriteConcern of type int64
`

 so in response it seems the `defaultWriteConcern` will always return as string